dist/common/scripts/scylla_io_setup

```diff
@@ -102,6 +103,28 @@ class scylla_cpuinfo:
         else:
             return len(self._cpu_data["system"])
 
+def configure_iotune_open_fd_limit(shards_count):
+    try:
+        fd_limits = resource.getrlimit(resource.RLIMIT_NOFILE)
+    except (OSError, ValueError) as e:
+        logging.warning("Could not get the limit of count of open file descriptors!")
+        logging.warning("iotune will proceed with the default limit. This may cause problems.")
+        return
+
+    soft_limit, hard_limit = fd_limits
+    required_fds_count = 8 * shards_count
```

Comment:
Let's add 500 here. We don't want some change in iotune that eats 30 file descriptors causing failures when shard_count = 2.

Reply:
Okay, I will change it to `(8 * shards_count) + 500`. This is the total number of needed file descriptors.

Btw. 8 is just an arbitrary value - I saw that iotune uses 6 FDs per shard. Therefore, I selected 8. It may be too small or become too small in the future.

Below there is a condition that was intended to prevent touching the limit when `8 * shards_count` was less than the default limit:
`if soft_limit < required_fds_count or hard_limit < required_fds_count:`

So, in the case when required number of FDs is lower than the default limit (I assumed 1024), then the code does not set it at all. This would prevent the error in the case of 2 shards and +30 FDs - still it is less than 1024.

Reply:
Fixed :+1:

Reply:
Let's run it with unlimited number instead. It's one-shot script, what's the point in taking care of its resource consumption?

Reply:
I tried to use `RLIM_INFINITY`, but on the target machine the hard limit was not set to unlimited. As the result, I saw raising an error.

> Raises [ValueError](https://docs.python.org/3/library/exceptions.html#ValueError) if an invalid resource is specified, if the new soft limit exceeds the hard limit, or if a process tries to raise its hard limit. Specifying a limit of [RLIM_INFINITY](https://docs.python.org/3/library/resource.html#resource.RLIM_INFINITY) when the hard or system limit for that resource is not unlimited will result in a [ValueError](https://docs.python.org/3/library/exceptions.html#ValueError). A process with the effective UID of super-user can request any valid limit value, including unlimited, but [ValueError](https://docs.python.org/3/library/exceptions.html#ValueError) will still be raised if the requested limit exceeds the system imposed limit.

Reference: https://docs.python.org/3/library/resource.html

Reply:
@xemul: I adjusted the code to set the limit as `max(precalculated_fds_count, hard_limit)`. It should make the code a little more robust.

For instance, on the target machine that I use, `ulimit -n -H` shows 1048576, which is large in comparison with the calculated count.